zerver/lib/actions.py

```diff
@@ -3737,10 +3737,14 @@ def user_info(um: UserMessage) -> Dict[str, Any]:
 
 
 def do_delete_message(user_profile: UserProfile, message: Message) -> None:
+    print(message)
     event = {
         'type': 'delete_message',
         'sender': user_profile.email,
-        'message_id': message.id}  # type: Dict[str, Any]
+        'message_id': message.id,
+        'is_stream_message': message.is_stream_message(),
+        'stream_id': message.recipient.type_id,
```

Comment:
This should be either `stream_id` or `recipient_user_ids`, depending on whether it's a stream or private message.